### PR TITLE
[enhancement](stream load pipe) using queryid or load id to identify stream load pipe instead of fragment instance id

### PR DESCRIPTION
**Pull request by yiguolei**
# Proposed changes

NewLoadStreamMgr already has pipe and other info. Do not need save the pipe into fragment state. and FragmentState should be more clear.

But this pr will change the behaviour of BE. 
I will pick the pr to doris 1.2.3 and add the load id to FE support. The user could upgrade from 1.2.3 to 2.x

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yiguolei** [comment]
./run buildall

**Comment by yiguolei** [comment]
./run buildall

**Comment by yiguolei** [comment]
./run p0

**Comment by yiguolei** [comment]
./run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java [hunk 1]
```diff
@@ -1405,6 +1402,7 @@ private void handleTransactionStmt() throws Exception {
                         .append(context.getTxnEntry().getTxnConf().getTxnId()).append("'").append("}");
                 context.getState().setOk(0, 0, sb.toString());
             } catch (Exception e) {
+                LOG.warn("Txn commit failed", e);
```
Comment (chenlinzhong) [inline fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java:1405]: LOG.warn("Txn commit failed, error={}", e.getMessage());

File: fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java [hunk 2]
```diff
@@ -623,10 +623,8 @@ public void execute(TUniqueId queryId) throws Exception {
             context.getState().setError(ErrorCode.ERR_UNKNOWN_ERROR, e.getMessage());
             throw e;
         } catch (UserException e) {
-            LOG.warn("", e);
             // analysis exception only print message, not print the stack
-            LOG.warn("execute Exception. {}, {}", context.getQueryIdentifier(),
-                                    e.getMessage());
+            LOG.warn("execute Exception. {}", context.getQueryIdentifier(), e);
```
Comment (chenlinzhong) [inline fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java:627]: LOG.warn("execute Exception. {},error={}", context.getQueryIdentifier(), e.getMessage());

File: fe/fe-core/src/main/java/org/apache/doris/qe/InsertStreamTxnExecutor.java [hunk 3]
```diff
@@ -82,6 +84,10 @@ public void beginTransaction(TStreamLoadPutRequest request) throws UserException
             }
         }
         txnConf.setFragmentInstanceId(tRequest.params.fragment_instance_id);
+        this.loadId = request.getLoadId();
```
Comment (chenlinzhong) [inline fe/fe-core/src/main/java/org/apache/doris/qe/InsertStreamTxnExecutor.java:87]: where set the loadid ?

Comment (chenlinzhong) [inline fe/fe-core/src/main/java/org/apache/doris/qe/InsertStreamTxnExecutor.java:87]: where set the request loadid ?

File: fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java (hunk 2 above)
Comment (chenlinzhong) [reply fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java:627]: why delete the execption message ?

**Review by chenlinzhong — APPROVED** [review]
LGTM

**Pull request merged**